src/tools/getdns_query.c

```diff
@@ -1619,6 +1619,14 @@ static void incoming_request_handler(getdns_context *context,
 		getdns_dict_destroy(response);
 }
 
+static void stubby_log(void *userarg, uint64_t system,
+    getdns_loglevel_type level, const char *fmt, va_list ap)
+{
+	(void)userarg; (void)system; (void)level;
+	(void) vfprintf(stderr, fmt, ap);
```

Comment:
It would be nice to be able to re-tag the message as STUBBY: instead of GETDNS_DAEMON: 
Something to think about in the stubby repo?